quetz/rest_models.py

```diff
@@ -7,11 +7,12 @@
 import uuid
 from datetime import date, datetime
 from enum import Enum
-from typing import Generic, List, Optional, TypeVar
+from typing import Annotated, Dict, Generic, List, Optional, TypeVar, Union
 
 from pydantic import BaseModel, ConfigDict, Field, field_validator, model_validator
 
 T = TypeVar('T')
+URLType = Annotated[str, Field(pattern="^(http|https)://.+")]
```

Comment:
quetz currently supports python versions as far back as `3.7`. `Annotated` is only available in 3.9+